### ISSUE TITLE
Step implementations stopped working with testcafe 1.8.6

### ISSUE THREAD

**Issue by Marketionist** [kiwigrid/gherkin-testcafe#65]
Hello @Lukas-Kullmann,

After update from testcafe 1.8.5 to 1.8.6 a blocking error appears while running tests with gherkin-testcafe 2.5.0:
```
1) Error: Step implementation missing for: I go to URL "http://localhost:8001/test1.html"

      Browser: Chrome 83.0.4103.61 / macOS 10.11.6

         173 |      if (isMatched) {
         174 |        return this._runStep(stepDefinition.code, testController, parameters, table);
         175 |      }
         176 |    }
         177 |
       > 178 |    throw new Error(`Step implementation missing for: ${step.text}`);
         179 |  }
         180 |
         181 |  _runStep(step, testController, parameters, table) {
         182 |    const markedFn = testRunTracker.addTrackingMarkerToFunction(testController.testRun.id, step);
         183 |

         at GherkinTestcafeCompiler._resolveAndRunStepDefinition
      (/Users/marketionist/Documents/Work/Github/testcafe-cucumber-steps/node_modules/gherkin-testcafe/src/compiler.js:178:11)
         at <anonymous>
      (/Users/marketionist/Documents/Work/Github/testcafe-cucumber-steps/node_modules/gherkin-testcafe/src/compiler.js:108:28)
         at $$testcafe_test_run$$lTksjLf56$$
      (/Users/marketionist/Documents/Work/Github/testcafe-cucumber-steps/node_modules/testcafe/src/api/test-run-tracker.js:76:16)
         at <anonymous>
      (/Users/marketionist/Documents/Work/Github/testcafe-cucumber-steps/node_modules/testcafe/src/api/wrap-test-function.js:17:28)
         at TestRun._executeTestFn
      (/Users/marketionist/Documents/Work/Github/testcafe-cucumber-steps/node_modules/testcafe/src/test-run/index.js:296:19)
         at TestRun.start
      (/Users/marketionist/Documents/Work/Github/testcafe-cucumber-steps/node_modules/testcafe/src/test-run/index.js:346:24)
```
See the Travis CI build with the error - https://travis-ci.org/github/Marketionist/testcafe-cucumber-steps/jobs/693703771

**Comment by Lukas-Kullmann** [comment]
Hey @Marketionist 
thanks for the bug report. I published 2.5.1 with the fix.

**Comment by Marketionist** [comment]
Re-tested - works ok with gherkin-testcafe@2.5.1, thanks for the fast fix @Lukas-Kullmann!
